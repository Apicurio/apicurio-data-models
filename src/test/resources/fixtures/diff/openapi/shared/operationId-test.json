{
	"openapi": "3.0.2",
	"info": {
	  "version": "1.0.0",
	  "title": "Swagger Petstore"
	},
	"paths": {
		"/tests": {
			"get": {
				"description": "Get list of tests",
				"responses": {
					"default": {
						"description": "..."
					}
				}
			},
			"post": {
				"operationId": "createTest"
			}
		},
		"/tests/{id}": {
			"put": {
				"operationId": "updateTest"
			}
		}
	}
}