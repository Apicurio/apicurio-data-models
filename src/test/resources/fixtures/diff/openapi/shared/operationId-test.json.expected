[OPERATION_ID_REMOVED] |DANGEROUS_CHANGE| :: Operation ID removed | /paths[/tests]/post
[OPERATION_ID_MODIFIED] |DANGEROUS_CHANGE| :: Operation ID modified | /paths[/tests/{id}]/put
[OPERATION_ID_ADDED] |NON_BREAKING| :: Operation ID added | /paths[/tests]/get
